packages/babel-helpers/src/helpers.js

```diff
@@ -588,28 +588,35 @@ helpers.interopRequireDefault = helper("7.0.0-beta.0")`
 `;
 
 helpers.interopRequireWildcard = helper("7.0.0-beta.0")`
+  var REQUIRE_WILD_CACHE = typeof WeakMap === 'function' ? new WeakMap() : undefined;
```

Comment:
This might have problems, in case `_interopRequireWildcard` is used before this variable declaraiton is run. To be safe, I would prefer that it is initialized inside the function:

```js
var REQUIRE_WILD_CACHE;

export default function _interopRequireWildcard(obj) {
  if (obj && obj.__esModule) {
    return obj;
  }

  if (REQUIRE_WILD_CACHE) {
    if (REQUIRE_WILD_CACHE.has(obj)) return REQUIRE_WILD_CACHE.get(obj);
  } else if (typeof WeakMap === "function") {
    REQUIRE_WILD_CACHE = new WeakMap();
  }

  ...
```